pkgs/tools/inputmethods/fcitx5/fcitx5-lua.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, fcitx5
+, lua5_3
+, gettext
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fcitx5-lua";
+  version = "5.0.0";
+
+  src = fetchFromGitHub {
+    repo = "fcitx";
+    owner = "fcitx5-lua";
+    rev = "${version}";
+    sha256 = "sha256-ABQS2DaOpkjyMla9V3CU87PlreRnB4KXNan2PjJ7vWU=";
+  };
+
+  # More hacks
+  preInstall = ''
+    sed -i "s#${fcitx5}#$out#g" src/addonloader/cmake_install.cmake
+    sed -i "s#${fcitx5}#$out#g" src/imeapi/cmake_install.cmake
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  buildInputs = [
+    fcitx5
+    lua5_3
```

Comment:
Is it possible to set the lua version to be selectable by the user？

Reply:
yes

Reply:
The Lua package to use can be overridden now.